src/components/TopRatedMovieCardList/TopRatedMovieCardList.tsx

```diff
@@ -0,0 +1,8 @@
+import MovieCardList from '../MovieCardList/MovieCardList';
+import { getTopRatedMovies } from '../../services/moviesAPI';
+
+const TopRatedMovieCardList: React.FC = () => {
+    return <MovieCardList title="Top Rated" fetchMovies={getTopRatedMovies} />;
+};
+
+export default TopRatedMovieCardList;
```

Comment:
Remove this component entirely

BUT

Create a new one that will map through SECTIONS const and create MovieLists